user/dao/dao.go

```diff
@@ -136,3 +155,22 @@ func (dao *DAO) DeleteUser(input DeleteUserInput) error {
 
 	return nil
 }
+
+// CreatePicture new picture in the datastore, returning the newly created picture
```

Comment:
Do we want the DAO to return the picture?

Reply:
I just did it for consistency, but agree we probably don't need it. Can remove it - what do you think?